daprdocs/content/en/reference/components-reference/supported-bindings/mysql.md

```diff
@@ -81,17 +87,37 @@ This component supports **output binding** with the following operations:
 - `query`
 - `close`
 
+### Parametrized queries
+
+This binding supports parametrized queries, which allow separating the SQL query itself from user-supplied values. The usage of parametrized queries is **strongly recommended** for security reasons, as they prevent [SQL Injection attacks](https://owasp.org/www-community/attacks/SQL_Injection).
+
+For example:
+
+```sql
+-- ❌ WRONG! Includes values in the query and is vulnerable to SQL Injection attacks.
+SELECT * FROM mytable WHERE user_key = 'something';
+
+-- ✅ GOOD! Uses parametrized queries.
+-- This will be executed with parameters ["something"]
+SELECT * FROM mytable WHERE user_key = ?;
+```
+
+> [Relevant XKCD](https://xkcd.com/327/)
```

Comment:
My .02. this is well-known enough, and humor helps getting important points across!

Reply:
That is funny and will be recorded in the history of this PR. Can we remove it to get this merged?